src/styles/utils/_functions.scss

```diff
@@ -0,0 +1,18 @@
+@for $date from 1 through 31 {
+  .calendar__cell:nth-child(#{$date})::before {
+    content: "#{$date}";
+  }
```

Comment:
maybe you shouldn't separate this part from styles in lines 22-25
it belongs to the same element